src/test/java/com/palantir/docker/compose/execution/RetryingDockerComposeShould.java

```diff
@@ -57,6 +57,17 @@ public void calls_up_on_the_underlying_docker_compose() throws IOException, Inte
     }
 
     @Test
+    public void calls_exec_on_the_underlying_docker_compose() throws IOException, InterruptedException {
+        DockerComposeExecOption options = new DockerComposeExecOption(new String[] {"-d", "-i"});
+        String container = "just a name";
+        DockerComposeExecArgument arguments = new DockerComposeExecArgument(new String[] {"ls", "-la"});
```

Comment:
If you make DockerComposeExecArgument and Options take a var args rather than an array then this will be a lot nicer.